client/state/index.js

```diff
@@ -119,6 +121,11 @@ if ( typeof window === 'object' ) {
 		require( './analytics/middleware.js' ).analyticsMiddleware,
 		require( './data-layer/wpcom-api-middleware.js' ).default,
 	);
+
+	//TEMPORARY AT FLOW FIX, NOT INTENDED FOR PROD
+	if ( config.isEnabled( 'automated-transfer' ) && config( 'env' ) !== 'production' ) {
```

Comment:
if I could request a pause on this, @blowery has redone this code in #10897 and that would be a good foundation for this if we waited (would also prevent merge conflict)